internal/distro/fedora/package_sets.go

```diff
@@ -238,6 +238,28 @@ func iotCommitPackageSet(t *imageType) rpmmd.PackageSet {
 		ps = ps.Append(rpmmd.PackageSet{Include: []string{"greenboot-default-health-checks"}})
 	}
 
+	switch t.arch.Name() {
+	case distro.X86_64ArchName:
+		ps = ps.Append(x8664IotCommitPackageSet(t))
+	case distro.Aarch64ArchName:
+		ps = ps.Append(aarch64IotCommitPackageSet(t))
+	default:
+		panic(fmt.Sprintf("unsupported arch: %s", t.arch.Name()))
+	}
+
+	if !common.VersionLessThan(t.arch.distro.osVersion, "38") {
+		ps = ps.Append(rpmmd.PackageSet{
+			Include: []string{
+				"fdo-client",
+				"fdo-owner-cli",
+				"ignition",
+				"ignition-edge",
```

Comment:
In the ps variable above we already have `ignition`, the one that is missing is `ignition-edge`